importlab/output.py

```diff
@@ -22,7 +22,7 @@ def format_file_node(import_graph, node, indent):
     """Prettyprint nodes based on their provenance."""
     f = import_graph.provenance[node]
     if isinstance(f, resolve.Direct):
-        out = f.path
+        out = '+ ' + f.short_path
```

Comment:
flake8: ./importlab/output.py:4:1: F401 'os' imported but unused
(It looks like I can't comment on unchanged lines...)